rule/max_public_structs.go

```diff
@@ -19,7 +20,7 @@ const defaultMaxPublicStructs = 5
 func (r *MaxPublicStructsRule) configure(arguments lint.Arguments) {
 	r.Lock()
 	defer r.Unlock()
-	if r.max == 0 {
+	if r.max != 0 {
```

Comment:
I'm worried by what the code will do with maxstruct set to 0

configure will be called again and again,no?

I mean someone who want to disable the rule, like what have been done a few days ago with file-max-lines